src/util/misc/hypot.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Returns the square root of the sum of squares of its arguments\
+ * Chrome implements `Math#hypot` with a calculation that affects precision so we hard code it as a util
+ * @see https://stackoverflow.com/questions/62931950/different-results-of-math-hypot-on-chrome-and-firefox
+ * @static
+ * @memberOf fabric.util
+ * @param {...number} values
+ * @return {number}
+ */
+export const hypot = (...values: number[]): number => {
+  let sumOfSquares = 0;
+  for (let i = 0; i < values.length; i++) {
+    sumOfSquares += values[i] * values[i];
+  }
+  return Math.sqrt(sumOfSquares);
+};
```

Comment:
do we ever need the  3 coordinate cases in fabricJS?
if no, can we please reduce this to `return Math.sqrt(x * x + y * y)` ?

Reply:
This is here because of the chrome impl of Math.hypot that causes needless headache.
So yes it is needed

Reply:
it is needed if you use it outside the bidimensional use case.
Otherwise the reduced 2d formula is equivalent since we aren't a polyfill library and we don't offere a precise math.hypot for nth dimension space.

My question indeed was, do we ever need the 3 or plus dimensions in hypotenuse?

Reply:
removed

Reply:
I want to use this example to point out a comunication issue
I ask you if between the calculation of projections, the 3+ dimension of hypot are used, if not to use the 2d version that is shorter.
You tell me that chrome has a suboptimal implementation, but that is written exactly over that file.
So i ask you gain the same thing, and you answer me with `removed`.
What is the process that brough you to the decision to remove the util? what are we using in place of the util now?

Reply:
The 2d version impl of Point.distanceFrom as we should have done from the beginning.

Reply:
I looked at it again. Gave another thought. Decided to hear your point out and look again with fresh eyes. That is the process. Then I remrmbered distanceFrom.

Reply:
To be clear I marked it as removed since I wrote about it in the review

Reply:
My bad for sure.

> You tell me that chrome has a suboptimal implementation, but that is written exactly over that file.

A lot of files and PRs so I don't remember it all.